app/lib/finder_schema.rb

```diff
@@ -6,15 +6,21 @@ def self.schema_names
     end
   end
 
-  attr_reader :base_path, :organisations, :format, :content_id, :editing_organisations
+  def self.load_schema_for(type)
+    JSON.parse(File.read(Rails.root.join("lib/documents/schemas/#{type}.json")))
+  end
+
+  attr_reader :base_path, :organisations, :editing_organisations, :taxons, :format, :content_id, :document_title
```

Comment:
Might want to consolidate the schema attribute reader to here.

Reply:
Good idea.